src/main/java/de/embl/cba/mobie/bookmark/BookmarksManager.java

```diff
@@ -108,7 +108,7 @@ public void saveCurrentSettingsAsBookmark () {
 		ArrayList<Bookmark> bookmarks = new ArrayList<>();
 		bookmarks.add(currentBookmark);
 
-		if (bookmarkNameAndLocation.location.equals(FileUtils.PROJECT) &&
+		if ( bookmarkNameAndLocation.location == FileLocation.Project &&
```

Comment:
I think it is better practice to use `.equals( )` instead of `==`.
For example for Strings `==` yields unexpected behaviour.